### ISSUE TITLE
Nan parameters

### ISSUE THREAD

**Issue by kkkkk001** [benedekrozemberczki/AttentionWalk#7]
Thanks for your pytorch code. I found that my parameters become Nan during training. 
Nan parameters include model.left_factors, model.right_factors, model.attention. All the entries of them become Nan during training. And also the loss.
I'm trying to find the reason. I would appreciate it if you could give me some help or hints.

**Comment by benedekrozemberczki** [comment]
Could you provide more details?

**Comment by kkkkk001** [comment]
I print the loss in the first 100 epochs. The file is attached below.
The loss decreased in the first 61 epochs as desired and became Nan at the 62-nd epoch.
[losslog.txt](https://github.com/benedekrozemberczki/AttentionWalk/files/5880577/losslog.txt)

**Comment by benedekrozemberczki** [comment]
What is the graph that you operate on? What are the hyperparameters?

**Comment by kkkkk001** [comment]
I'm trying on BlogCatalog, an undirected graph. The hypermeters stay as default except for dimension. Dimension is changed into 256.

**Comment by benedekrozemberczki** [comment]
Is that a connected graph?

**Comment by kkkkk001** [comment]
Yes, it is connected.

**Comment by benedekrozemberczki** [comment]
Can you reduce the dimension?

**Comment by kkkkk001** [comment]
Is this caused by dimension?

**Comment by benedekrozemberczki** [comment]
No most probably an overflow in scores which is exponentiated somewhere.
